slabs.c

```diff
@@ -1189,7 +1214,10 @@ static void *slab_rebalance_thread(void *arg) {
         } else if (was_busy) {
             /* Stuck waiting for some items to unlock, so slow down a bit
              * to give them a chance to free up */
-            usleep(1000);
+            usleep(backoff_timer);
```

Comment:
curious; did you try an additive vs multiplicative backoff in your tests?

Reply:
I did, but got more busy items.